CSS outline follows border-radius in Firefox 88

mirunacurtean opened the pull request:
A checklist to help your pull request get merged faster:
- [x] Summarize your changes

Updating FF and FF android columns version and adding notes "From Firefox 88 onwards, the standard outline property will follow the shape of border-radius."

- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)

https://bugzilla.mozilla.org/show_bug.cgi?id=1694146

- [x] Link to related issues or pull requests, if any
- Fixes https://github.com/mdn/content/issues/3453

ddbeck commented on css/properties/outline-offset.json:
```diff
@@ -16,11 +16,18 @@
-            "firefox_android": {
-              "version_added": "4"
-            },
+              "version_added": "88",
+              "notes": "From Firefox 88 onwards, the standard outline property will follow the shape of border-radius."
```
```suggestion
              "version_added": "1.5",
              "notes": "Before Firefox 88, an outline does not follow the shape of <code>border-radius</code>."
```

ddbeck reviewed (changes requested):
Thanks for this PR, @mirunacurtean. Several suggestions for you, with explanations in `outline.json`. Let me know if you have any questions!

mirunacurtean commented on css/properties/outline-color.json:
```diff
@@ -17,17 +17,24 @@
             "firefox": [
               {
-                "version_added": "1.5"
+                "version_added": "88",
+                "notes": "From Firefox 88 onwards, the standard outline property will follow the shape of border-radius."
```
Alright, At the moment I don't know how to undo just that so I'll just update those rows to contain the previous information and push this out along with all the other suggested changes for this PR. Thanks!

mirunacurtean commented on css/properties/outline.json:
```diff
@@ -17,18 +17,24 @@
-                "version_added": "1.5",
-                "notes": "Firefox includes absolutely positioned elements inside the outline (see <a href='https://bugzil.la/687311'>bug 687311</a>)."
+                "version_added": "88",
+                "notes": "From Firefox 88 onwards, the standard outline property will follow the shape of border-radius."
```
I'm taking the expert opinion here into account :) I've updated this based on the suggestions.

mirunacurtean commented on css/properties/outline.json:
```diff
@@ -17,18 +17,24 @@
+            "firefox_android": [
+              {
+                "version_added": "88",
+                "notes": "From Firefox 88 onwards, the standard outline property will follow the shape of border-radius."
```
Done in next commit.

mirunacurtean commented:
@ddbeck I hope I incorporated all suggestions successfully after the latest corrections (initially I used the same version_added for FF desktop and android and that was incorrect, so I've updated all in all files). Checks are still running so fingers crossed. 
Sorry for the messiness of these updates!

ddbeck reviewed (changes requested):
There are a few things left to fix here:

1. Some mis-indentation is causing CI to fail. If you run `npm run fix` and commit the results, it should correct this automatically.
2. Please delete `yarn.lock`. We don't use it.
3. I left a line comment highlight a spot I missed on the last go-around.

Thanks!

mirunacurtean commented on css/properties/outline-offset.json:
```diff
@@ -16,10 +16,12 @@
             },
             "firefox": {
-              "version_added": "1.5"
+              "version_added": "88",
```
Applied in next commit.

mirunacurtean commented:
@ddbeck I think this is finally ready for another look.

ddbeck reviewed (approved):
Looks good. Thank you! 🎉
